components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtilsTest.java

```diff
@@ -48,6 +62,37 @@ private void setMockedConfigurationFacade() {
         when(ConfigurationFacade.getInstance()).thenReturn(mockedConfigurationFacade);
     }
 
+    private void mockFrameworkServiceComponent() {
```

Comment:
Add a new line after the method signature.
Fix in other relevant places.

Reply:
Already fixed with commit baf49d4b0e85432467ae5848dcf443c0032ce5c9 in https://github.com/wso2/carbon-identity-framework/pull/3509 PR